app/controllers/api/v2/registration_controller.rb

```diff
@@ -34,7 +34,8 @@ def global
       param :name, String, required: true
       param :location_id, :number, required: true
       param :organization_id, :number, required: true
-      param :ip, String, desc: N_("not required if using a subnet with DHCP proxy")
+      param :ip, String, desc: N_("IPv4 address, not required if using a subnet with DHCP proxy")
+      param :ip6, String, desc: N_("IPv6 address, not required if using a subnet with DHCP proxy")
       param :mac, String, desc: N_("required for managed host that is bare metal, not required if it's a virtual machine")
```

Comment:
I didn't really read through the first iteration, but I'd drop the `not required if it's a virtual machine`. The MAC is really irrelevant when using a compute resource, but you can't do that via the registration controller.

Reply:
Yeah, true. We don't to any dhcp orchestration while registering a host. The mac address is not really important.